docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -182,71 +210,149 @@ For example, to turn off asynchronous eviction:
+
+
 ## Manage Data in Alluxio
 
-TODO(Zac): Explain Alluxio-specific concepts in storage which is unique to Alluxio.
+<!-- ################## -->
+<!-- TODO(Zac): Explain Alluxio-specific concepts in storage which is unique to Alluxio.
 We want to cover
 - "free": remove data from Alluxio space but the same data will still be available in UFS if
 persisted
 - "load": preload data from UFS to Alluxio
 - "persist"
-- "ttl": this section is already added.
+- "ttl": this section is already added. -->
+<!-- ################## -->
+
+Within an Alluxio cluster there are concepts related to storage which must be understood in order
+to properly utilize the available resources. Users should understanding the following:
+
+- **free**: Freeing data in Alluxio means removing data from the Alluxio cache, but not the
+underlying UFS. Data is still available to users after a free, but performance may be degraded
+to those who attempt to access a file after it has been freed from Alluxio.
+- **load**: Loading data into Alluxio means moving it from a UFS into Alluxio cache. If
+Alluxio is using memory-based storage, users will likely see I/O performance improvements after
+loading data into Alluxio.
+- **persist**: In the context of Alluxio persisting means writing data which may or may not have
+been modified within Alluxio storage back to a UFS. By writing data back to a UFS data is
+guaranteed not to be lost if an ALluxio node goes down.
+- **TTL (Time to Live)**: Files and directories within Alluxio space have a TTL property which can
+be set in order to effectively manage Alluxio space. By setting a TTL Alluxio remove old data from
+the Alluxio cache after a period of time in order to make room for new data.
+
+### Freeing Data from Alluxio Storage
+
+In order to manually free data in Alluxio, you can use the `./bin/alluxio` filesystem command
+line interface.
+
+```bash
+$ ./bin/alluxio fs free ${PATH_TO_UNUSED_DATA}
+```
+
+This will remove data from Alluxio storage. but if the data is persisted to a UFS, it will still be
+accessible. For more information refer to the
+[command line interface documentation]({{site.baseurl}}{% link en/advanced/Command-Line-Interface.md %}#free)
+
+It should be noted that a user typically should not need to manually free data from Alluxio as the
+configured [eviction policy](#eviction-policies) will take care of removing unused or old data.
+
+### Loading Data into Alluxio Storage
+
+There are a couple ways that a user may move data into Alluxio storage. If the data is already in a
+UFS, then the user may use
+[`alluxio fs load`]({{site.baseurl}}{% link en/advanced/Command-Line-Interface.md %}#load)
+
+```bash
+$ ./bin/alluxio fs load ${PATH_TO_FILE}
+```
+
+If there is data within the local filesystem, the
+[command `copyFromLocal`]({{site.baseurl}}{% link en/advanced/Command-Line-Interface.md %}#copyfromlocal)
+which can be used to load data into Alluxio storage, but will not persist the data to a UFS.
```

Comment:
The write type controls whether the data is persisted to the UFS. `MUST_CACHE` won't persist to UFS, `CACHE_THROUGH` and `THROUGH` will